zerver/views/video_calls.py

```diff
@@ -268,9 +262,13 @@ def join_bigbluebutton(request: HttpRequest, bigbluebutton: str = REQ()) -> Http
     join_params = urlencode(
         {
             "meetingID": bigbluebutton_data["meeting_id"],
-            # We use the moderator password here to grant ever user
-            # full moderator permissions to the bigbluebutton session.
-            "password": bigbluebutton_data["password"],
```

Comment:
What's the thinking around removing the `password` field? I'm not sure what it's implications are for the security model here. @strifel FYI.

Reply:
I explained the reasoning [here](https://chat.zulip.org/#narrow/stream/101-design/topic/audio-only.20calls/near/1692721) on CZO. Let me know if there's anything else I should clarify!

Reply:
This seems fine to me.
We might only want to clarify that we only support BigBlueButton 2.4 and newer.